### PR TITLE
refactor(cdci): improves workflows code style (#206)

### PR DESCRIPTION
**Pull request by ygrishajev**
This PR is an example for the [referenced issue](https://github.com/akash-network/support/issues/206). It's not intended to be merged at this point until there's an agreement with the team made.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -8,30 +8,43 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
+      - name: Checkout Code
```
Comment (troian) [inline .github/workflows/release.yaml:11]: lets keep word as sentence where only first letter is capital

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -8,30 +8,43 @@ jobs:
...
+
+      - name: Install Changelog Generator
         run: go install github.com/git-chglog/git-chglog/cmd/git-chglog@${{ env.GIT_CHGLOG_VERSION }}
-      - name: Generate changelog
+
+      - name: Generate Changelog
         run: make changelog
-      - uses: cb80/delrel@latest
+
+      - name: Delete Previous Manual Release
```
Comment (troian) [inline .github/workflows/release.yaml:40]: Delete previous release if exists

File: .github/workflows/release.yaml (hunk 1 above)
Comment (ygrishajev) [reply .github/workflows/release.yaml:11]: I don't mind. My reasoning is that this is a name or a title which generally has every word capitalized

Comment (ygrishajev) [reply .github/workflows/release.yaml]: done

File: .github/workflows/release.yaml (hunk 2 above)
Comment (ygrishajev) [reply .github/workflows/release.yaml]: done

**Pull request merged**